### ISSUE TITLE
Incorrect "Account has run out of space" error

### ISSUE THREAD

**Issue by teleshoes** [conformal/cyphertite#92]
this error is also affecting me. unlimited account, happens every time i try to upload anything.

REALLY REALLY frustrating, as i was at 94% of a 400GiB upload.
once it is fixed, it will take many hours to get back up to 94%, even with deduplication.

**Comment by teleshoes** [comment]
magically fixed

**Comment by teleshoes** [comment]
magically unfixed...

**Comment by csmiken** [comment]
We had a server issue that incorrectly reported to some users that they were out of space. We had to perform some maintenance on the servers because of this and I apologize that this may have interrupted your backup. The issues have been resolved at this point and I can verify that things are running properly. We appreciate your business and do not expect this weekend's issue to return. If you restart your backup, the deduplicaion method we use should quickly return to where your backup left off. This is because Cyphertite will already know that the majority of the chunks of your data are already sitting on our servers. It will only send the remaining ones and then complete the backup properly.
